infrastructure/docker/services/php-base/Dockerfile

```diff
@@ -5,14 +5,17 @@ RUN apt-get update \
         ca-certificates \
         gnupg \
     && echo "deb https://packages.sury.org/php buster main" > /etc/apt/sources.list.d/sury.list \
-    && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B188E2B695BD4743 \
-    && apt-get update
+    && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B188E2B695BD4743
 
-RUN apt-get install -y --no-install-recommends \
-        procps
+RUN apt-get update \
```

Comment:
Do we need to make an update on each RUN command ?

Reply:
Yes, to prevent the issue I got. (For external readers, I'm sorry because I shared my issue only on a private slack)

But to summarize quickly:

Each layer can are cached based on the context (in our case it's the command to run)
If you run docker build now, all layer will be cached with today information. So APT cache will also have information of today.

In one year (I take a very long period on purpose) you may add `php7.4-amqp` to your stack.
So the first cached layers will be used. Since the `update` is done in a previous & cached layer,  APT will have an outdated cache.  So many packet will be missing.

That's why I added  that on each layer: To always have an up to date apt cache on each steap.